### ISSUE TITLE
[Bug]: [perf-nightly]Milvus load collection failed ,raise an error"failed to Deserialize index, invalid binary set"

### ISSUE THREAD

**Issue by jingkl** [milvus-io/milvus#26178]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:master-20230807-07f08daf
- Deployment mode(standalone or cluster):standalone
- MQ type(rocksmq, pulsar or kafka):    rocksmq
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

argo info：
deploy_config
fouramf-server-standalone-8c16m
case_params
fouramf-client-glove1m-concurrent-ivfflat
case_name
test_concurrent_locust_custom_parameters

server:

```
 I0807 09:10:58.257258     547 request.go:665] Waited for 1.161887155s due to client-side throttling, not priority and fairness, request: GET:https://kubernetes.default.svc.cluster.local/apis/authorization.k8s.io/v1?timeout=32s
NAME                                                              READY   STATUS      RESTARTS        AGE     IP              NODE         NOMINATED NODE   READINESS GATES
perf-grow-ng5p5-5-5-3520-etcd-0                                   1/1     Running     0               13m     10.104.4.221    4am-node11   <none>           <none>
perf-grow-ng5p5-5-5-3520-milvus-standalone-d9f98877-qv4jt         1/1     Running     0               13m     10.104.19.132   4am-node28   <none>           <none>
perf-grow-ng5p5-5-5-3520-minio-7fd8887946-4btlx                   1/1     Running     0               13m     10.104.4.217    4am-node11   <none>           <none> (cli_client.py:132)
```

client log:

```
2023-08-07 09:10:56,720 - ERROR - fouram]: RPC error: [get_loading_progress], <MilvusException: (code=1, message=show collection failed: legacy error code:1, reason: UnexpectedError: Assert "failed to Deserialize index, invalid binary set" at /go/src/github.com/milvus-io/milvus/internal/core/src/index/VectorMemIndex.cpp:89
)>, <Time:{'RPC start': '2023-08-07 09:10:56.718710', 'RPC error': '2023-08-07 09:10:56.720218'}> (decorators.py:108)
[2023-08-07 09:10:56,721 - ERROR - fouram]: RPC error: [wait_for_loading_collection], <MilvusException: (code=1, message=show collection failed: legacy error code:1, reason: UnexpectedError: Assert "failed to Deserialize index, invalid binary set" at /go/src/github.com/milvus-io/milvus/internal/core/src/index/VectorMemIndex.cpp:89
)>, <Time:{'RPC start': '2023-08-07 09:00:45.955955', 'RPC error': '2023-08-07 09:10:56.721103'}> (decorators.py:108)
[2023-08-07 09:10:56,721 - ERROR - fouram]: RPC error: [load_collection], <MilvusException: (code=1, message=show collection failed: legacy error code:1, reason: UnexpectedError: Assert "failed to Deserialize index, invalid binary set" at /go/src/github.com/milvus-io/milvus/internal/core/src/index/VectorMemIndex.cpp:89
)>, <Time:{'RPC start': '2023-08-07 09:00:45.759520', 'RPC error': '2023-08-07 09:10:56.721235'}> (decorators.py:108)
[2023-08-07 09:10:56,722 - ERROR - fouram]: (api_response) : <MilvusException: (code=1, message=show collection failed: legacy error code:1, reason: UnexpectedError: Assert "failed to Deserialize index, invalid binary set" at /go/src/github.com/milvus-io/milvus/internal/core/src/index/VectorMemIndex.cpp:89
)> (api_request.py:53)
[2023-08-07 09:10:56,722 - ERROR - fouram]: [CheckFunc] load request check failed, response:<MilvusException: (code=1, message=show collection failed: legacy error code:1, reason: UnexpectedError: Assert "failed to Deserialize index, invalid binary set" at /go/src/github.com/milvus-io/milvus/internal/core/src/index/VectorMemIndex.cpp:8
```

### Steps To Reproduce

```markdown
1. create a collection 
        2. build ivf flat index on vector column
        3. insert 1m 768d of vectors
        4. flush collection
        5. build index on vector column with the same parameters
        6. count the total number of rows
        7. load collection—》raise errors
```

### Anything else?

[2023-08-07 08:59:23,906 -  INFO - fouram]: [check_params] scene_concurrent_locust required params: {'dataset_params': {'metric_type': 'IP', 'dim': 200, 'dataset_name': 'glove', 'dataset_size': '1m', 'ni_per': 10000}, 'collection_params': {'other_fields': []}, 'load_params': {}, 'query_params': {}, 'search_params': {}, 'index_params': {'index_type': 'IVF_FLAT', 'index_param': {'nlist': 1024}}, 'concurrent_params': {'concurrent_number': [1, 20, 50, 100], 'during_time': 3600, 'interval': 20}, 'concurrent_tasks': [{'type': 'search', 'weight': 1, 'params': {'nq': 1, 'top_k': 1, 'search_param': {'nprobe': 16}, 'random_data': True}}]} (params_check.py:31)

**Comment by jiaoew1991** [comment]
/assign @yah01 
/unassign 

Is it similar to the recent fix?

**Comment by yah01** [comment]
> /assign @yah01 /unassign
> 
> Is it similar to the recent fix?

Looks like the fix doesn't work, checking it

**Comment by yah01** [comment]
/assign @jingkl 
fixed by #26190

**Comment by jingkl** [comment]
deploy_config
fouramf-server-standalone-8c16m
case_params
fouramf-client-glove1m-concurrent-ivfflat
other_params
case_name
test_concurrent_locust_custom_parameters

image：master-20230811-9489e140
already fix
